concert/helpers.py

```diff
@@ -299,7 +299,13 @@ async def get_state_from_awaitable(awaitable) -> str:
     task = asyncio.ensure_future(awaitable)
     if not task.done():
         return 'running'
-    elif task.exception() is None:
-        return 'standby'
     else:
-        return 'error'
+        try:
+            exc = task.exception()
+        except asyncio.CancelledError:
+            # That's the only exception type left which can be raised
+            return 'error'
```

Comment:
Here it's up for discussion, I can find arguments for making this 'standby' or a completely new state 'cancelled'. If it won't be standby we need to update the `check` targets of `Experiment` and `Acquisition` `run` methods.

Reply:
I would prefere 'cancelled'. 
I can directly add the new state in https://github.com/ufo-kit/concert/pull/504 to the checks.